File: tugas_nomor1.c
#include <stdio.h>

int main() {
    float suhuCel, reamur, farenheit;

    printf("Program Konversi Suhu Celsius ke Reamur dan Farenheit\n");
    printf("------------------------------------------------------\n");

    printf("Masukkan suhu dalam Celsius : ");
    scanf("%f", &suhuCel);

    reamur = suhuCel * (4.0/5.0);
    farenheit = (suhuCel * 9.0/5.0) + 32;

    printf("\nHasil Konversi:\n");
    printf("Reamur     : %.2f\n", reamur);
    printf("Farenheit  : %.2f\n", farenheit);

    return 0;
}

File: tugas_nomor1.pas
program KonversiSuhu;

uses crt;

var
  suhuCel, reamur, fahrenheit : real;

begin
  clrscr;

  writeln('Program Konversi Suhu Celsius ke Reamur dan Fahrenheit');
  writeln('------------------------------------------------------');
  write('Masukkan suhu dalam Celsius : ');
  readln(suhuCel);

  reamur := suhuCel * (4/5);
  fahrenheit := (suhuCel * 9/5) + 32;

  writeln;
  writeln('Hasil Konversi:');
  writeln('Reamur     : ', reamur:0:2);
  writeln('fahrenheit  : ', fahrenheit:0:2);

  readln;
end.